runner/src/lib.rs

```diff
@@ -72,7 +72,9 @@ fn run_module(module: &Module, entry_point: Option<&str>) -> Result<(), String>
         .create_jit_execution_engine(OptimizationLevel::None)
         .map_err(|e| e.to_string())?;
 
-    bind_functions(module, &execution_engine)?;
+    if !bind_functions(module, &execution_engine)? {
+        println!("WARNING: Mix of legacy and current output recording functions found. Please update your tooling.");
+    }
```

Comment:
We should undo this change along with the bool return of bind_functions with bool. The code that returns bool should return Err() with the message here.

Reply:
Updated!